components/service/glean/metrics.yaml

```diff
@@ -0,0 +1,96 @@
+# This file defines the metrics that are recorded by glean telemetry. They are
+# automatically converted to Kotlin code at build time using the `glean_parser`
+# PyPI package.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+baseline:
+  # sessions:
```

Comment:
These are commented out since we don't have the underlying metric types implemented yet.

Reply:
Let's add this as a comment before the probes. Also mention the bug numbers they depend on (e.g. 1497894)